aci/resource_aci_snmpcommunityp_deprecated.go

```diff
@@ -0,0 +1,176 @@
+package aci
```

Comment:
I suppose we are going to deprecate resource aci_vrf_snmp_context_community, it's confusing to me while using resource_aci_snmpcommunityp_deprecated as the resource file name.
maybe we want to use something like resource_aci_vrf_snmp_contextcommunity_deprecated

Reply:
all the resource file names are the Class names in this case both aci_vrf_snmp_context_community and aci_snmp_community uses the class snmpCommunityP so that is why I added the _deprecated to the file to understand which one to remove in the next major release.